### PR TITLE
Cmake Placement

### PR DESCRIPTION
**Pull request by ckissane**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -54,7 +54,7 @@ task invokeCMake(type: Exec) {
     workingDir 'src/main/cpp'
 }
 
-task buildDll(type: Exec, dependsOn: invokeCMake) {
+task buildDll(type: Exec){//, dependsOn: invokeCMake) {
```
Comment (DethRaid) [inline build.gradle]: Just remove the code

File: src/main/cpp/render/nova_renderer.cpp [hunk 2]
```diff
@@ -381,7 +383,7 @@ namespace nova {
     void nova_renderer::begin_frame() {
         LOG(TRACE) << "Beginning frame";
         cur_swapchain_image_index = render_context::instance.device.acquireNextImageKHR(render_context::instance.swapchain,
-                                                                               std::numeric_limits<uint32_t>::max(),
+                                                                               UINT32_MAX,
```
Comment (DethRaid) [inline src/main/cpp/render/nova_renderer.cpp:386]: Would rather undef max in this file

**Review by DethRaid — CHANGES_REQUESTED** [review]

File: src/main/cpp/CMakeSettings.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
+{
+  // See https://go.microsoft.com//fwlink//?linkid=834763 for more information about this file.
```
Comment (skmedix) [inline src/main/cpp/CMakeSettings.json:2]: `"_comment": "See https://go.microsoft.com//fwlink//?linkid=834763 for more information about this file.",`

File: setup.gradle [hunk 4]
```diff
@@ -207,6 +207,7 @@ task initSubmodules(dependsOn: cleanupMcp) << {
     }
 }
 
+
```
Comment (skmedix) [inline setup.gradle:210]: A new line is not necessary here

**Review by skmedix — CHANGES_REQUESTED** [review]

File: src/main/cpp/CMakeSettings.json [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+      "configurationType": "RelWithDebInfo",
+      "inheritEnvironments": [ "msvc_x64_x64" ],
+      "buildRoot": "${env.USERPROFILE}\\CMakeBuilds\\${workspaceHash}\\build\\${name}",
+      "installRoot": "${env.USERPROFILE}\\CMakeBuilds\\${workspaceHash}\\install\\${name}",
+      "cmakeCommandArgs": "-DCMAKE_BUILD_TYPE=RELEASE",
+      "buildCommandArgs": "-v",
+      "ctestCommandArgs": ""
+    }
+  ]
+}
```
Comment (DethRaid) [inline src/main/cpp/CMakeSettings.json:49]: Wait why did you commit this file

Don't commit this file

**Review by DethRaid — CHANGES_REQUESTED** [review]

**Pull request closed without merging**